make/photon/prepare/utils/db.py

```diff
@@ -15,4 +15,5 @@ def prepare_db(config_dict):
     render_jinja(
         db_env_template_path,
         db_conf_env,
-        harbor_db_password=config_dict['harbor_db_password'])
+        harbor_db_password=config_dict['harbor_db_password'],
+        harbor_db_max_connections=config_dict['harbor_db_max_open_conns'])
```

Comment:
Can we refrain from allowing user to set this in `harbor.yml`?

If it's set to a very large number I believe the `shared_buffer` must change accordingly.

I suggest for v2.1 and v2.0.2 we just hard code it in env of `harbor-db` with a relatively larger number, say 1000

Reply:
Changed to max_connections in Postgres to 1024 and max_open_conns to 1000 in the core.